src/extensibility/ExtensionManagerDialog.js

```diff
@@ -459,12 +459,14 @@ define(function (require, exports, module) {
             });
 
             // Filter the views when the user types in the search field.
+            var searchTimeOut = 0;
             $dlg.on("input", ".search", function (e) {
+                clearTimeout(searchTimeOut);
                 var query = $(this).val();
-                views.forEach(function (view) {
-                    view.filter(query);
+                searchTimeOut = setTimeout(function () {
+                    views[_activeTabIndex].filter(query);
                     $modalDlg.scrollTop(0);
-                });
+                }, 200);
```

Comment:
I tried those values, but there is still interruption in typing if it is 100 ms or 50 ms. I tried higher values also (like 150), but there is still interruption in typing, 200 ms worked just fine for me.